INSTALL

```diff
@@ -47,40 +45,33 @@ If you are on Mac OSX you can probably install matplotlib binaries using the
 standard Python installation program `pip <https://pypi.python.org/pypi/pip>`_.
 See :ref:`install_osx_binaries`.
 
+.. _installing_windows:
 
 Windows
 -------
 
 If you don't already have Python installed, we recommend using
 one of the `scipy-stack compatible Python distributions
 <http://www.scipy.org/install.html>`_ such as WinPython, Python(x,y),
-Enthought Canopy, or Continuum Anaconda, which have matplotlib and
-many of its dependencies, plus other useful packages, preinstalled.
-
-For `standard Python <http://www.python.org/download/>`_ installations
-you will also need to install compatible versions of
-`setuptools <https://pypi.python.org/pypi/setuptools/>`_,
-`numpy <https://pypi.python.org/pypi/numpy/>`_,
-`python-dateutil <https://pypi.python.org/pypi/python-dateutil/>`_,
-`pytz <https://pypi.python.org/pypi/pytz>`_,
-`pyparsing <https://pypi.python.org/pypi/pyparsing/>`_, and
-`cycler <https://pypi.python.org/pypi/Cycler>`_
-in addition to
-`matplotlib <http://pypi.python.org/pypi/matplotlib/>`_.
-
-For Python 3.5 the `Visual C++ Redistributable for Visual Studio 2015
-<http://www.microsoft.com/en-us/download/details.aspx?id=48145>`_
-needs to be installed.
-In case Python 2.7 to 3.4 are not installed for all users (not the default),
+Enthought Canopy, or Continuum Anaconda, which have matplotlib and many
+of its dependencies, plus other useful packages, preinstalled.
+
+For `standard Python <http://www.python.org/download/>`_ installations,
+install matplotlib using `pip <https://pypi.python.org/pypi/pip/>`_::
+
+    python -m pip install -U pip setuptools
+    python -m pip install matplotlib
+
+In case Python 2.7 or 3.4 are not installed for all users (not the default),
```

Comment:
double negative makes it unclear as whether the default behavior is that they're install for users or not. Rephrase to _default is installed for all users_